swarm_test.go

```diff
@@ -190,6 +190,7 @@ func TestNewSwarm(t *testing.T) {
 	}
 }
 
+//TODO: is this still needed here, as we are validating all these fields now in SWAP
```

Comment:
Well what do you think? :)

You moved it to the other place, so just ask yourself if the new tests cover everything the old ones did.

Reply:
`with swap enabled and default network ID` is the only relevant test. Thanks for your comment!